more_itertools/more.py

```diff
@@ -2823,9 +2823,9 @@ class seekable:
         >>> it.relative_seek(-3)  # Source is at '6', we move back to '3'
         >>> next(it)
         '3'
-        >>> it.relative_seek(-3)  # Source is still at '6', we move back to '3'
+        >>> it.relative_seek(-3)  # Source is at '3', we move back to '1'
```

Comment:
Should `Source is at '4'`, right? That's the one that would be emitted if `next()` were called.

Reply:
You are right, should be '4'

Reply:
Fixed now